src/manage_acl.c

```diff
@@ -1143,13 +1143,14 @@ acl_where_owned_user (const char *user_id, const char *user_sql,
           else if (strcmp (type, "result") == 0)
             permission_clause
              = g_strdup_printf ("%s"
-                                " OR results%s.task IN"
+                                " OR EXISTS"
                                 " (SELECT id FROM %spermissions_subject"
```

Comment:
Feels like this should be IN to match the other cases. In 28054e8dff5328572e649f2d0562e746acc86451 in PR  #2118 @timopollmeier changed the EXISTS to IN, but he forgot to change `id` to `resource`. Maybe messed up his performance measurements, but maybe he was measuring the other cases.